apps/ledger-live-desktop/src/renderer/screens/recover/Player.tsx

```diff
@@ -79,12 +79,12 @@ export default function RecoverPlayer({
       theme,
       lang: locale,
       availableOnDesktop,
-      deviceId: state?.deviceId,
+      deviceId: device?.modelId,
```

Comment:
I'd keep the `deviceId` from the `state` just in case because I think we use that for the onboarding section IIRC
```suggestion
      deviceId: state?.deviceId || device?.modelId,
```